test/e2e/scoped_client_test.go

```diff
@@ -106,35 +106,36 @@ var _ = Describe("Scoped Client", func() {
 			return
 		}
 
-		if tt.grant != nil {
-			cleanupPerm := tt.grant(sa.GetNamespace(), sa.GetName())
+		if testParam.grant != nil {
+			cleanupPerm := testParam.grant(sa.GetNamespace(), sa.GetName())
 			defer cleanupPerm()
 		}
 
 		// We expect to get scoped client instance(s).
 		kubeclientGot, crclientGot, dynamicClientGot, errGot := strategy.AttenuateClient(getter)
-		require.NoError(GinkgoT(), errGot)
-		require.NotNil(GinkgoT(), kubeclientGot)
-		require.NotNil(GinkgoT(), crclientGot)
+		Expect(errGot).ToNot(HaveOccurred())
+		Expect(kubeclientGot).ToNot(BeNil())
+		Expect(crclientGot).ToNot(BeNil())
 
 		_, errGot = kubeclientGot.KubernetesInterface().CoreV1().ConfigMaps(namespace).Get(context.TODO(), genName("does-not-exist-"), metav1.GetOptions{})
-		require.Error(GinkgoT(), errGot)
-		tt.assertFunc(errGot)
+		Expect(errGot).To(HaveOccurred())
+		testParam.assertFunc(errGot)
 
 		_, errGot = crclientGot.OperatorsV1alpha1().CatalogSources(namespace).Get(context.TODO(), genName("does-not-exist-"), metav1.GetOptions{})
-		require.Error(GinkgoT(), errGot)
-		tt.assertFunc(errGot)
+		Expect(errGot).To(HaveOccurred())
+		testParam.assertFunc(errGot)
 
 		gvr := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "ConfigMap"}
 		_, errGot = dynamicClientGot.Resource(gvr).Namespace(namespace).Get(context.TODO(), genName("does-not-exist-"), metav1.GetOptions{})
-		require.Error(GinkgoT(), errGot)
-		tt.assertFunc(errGot)
+		Expect(errGot).To(HaveOccurred())
+		testParam.assertFunc(errGot)
 	}, tableEntries...)
 })
 
-func waitForServiceAccountSecretAvailable(t GinkgoTInterface, client operatorclient.ClientInterface, namespace, name string) *corev1.ServiceAccount {
+func waitForServiceAccountSecretAvailable(client operatorclient.ClientInterface, namespace, name string) *corev1.ServiceAccount {
 	var sa *corev1.ServiceAccount
-	err := wait.Poll(5*time.Second, time.Minute, func() (bool, error) {
+
+	Eventually(func() (bool, error) {
```

Comment:
If this fails, the failure stack trace will start from here instead of the place that is calling `waitForServiceAccountSecretAvailable`. Gomega provides [EventuallyWithOffset](https://pkg.go.dev/github.com/onsi/gomega?tab=doc#EventuallyWithOffset) to remove helper functions from the error stack trace.